docs/src/pages/plugins/broadcastQueryClient.md

```diff
@@ -0,0 +1,59 @@
+---
+id: broadcastQueryClient
+title: broadcastQueryClient (Experimental)
+---
+
+> VERY IMPORTANT: This utility is currently in an experimental stage. This means that breaking changes will happen in minor AND patch releases. Use at your own risk. If you choose to rely on this in production in an experimental stage, please lock your version to a patch-level version to avoid unexpected breakages.
+
+`broadcastQueryClient` is a utility for broadcasting and syncing the state of your queryClient between browser tabs/windows with the same origin.
+
+## Installation
+
+This utility comes packaged with `react-query` and is available under the `react-query/broadcastQueryClient-experimental` import.
+
+## Usage
+
+Import the `broadcastQueryClient` function, and pass it your `QueryClient` instance, and optionally, set a `broadcastChannel`.
+
+```ts
+import { broadcastQueryClient } from 'react-query/broadcastQueryClient-experimental'
+
+const queryClient = new QueryClient()
+
+broadcastQueryClient({
+  queryClient,
+  broadcastChannel: 'my-app',
+})
+```
+
+## API
+
+### `broadcastQueryClient`
+
+Pass this function a `QueryClient` instance and optionally, a `broadcastChannel`.
+
+```ts
+broadcastQueryClient({ queryClient, broadcastChannel })
+```
+
+### `Options`
+
+An object of options:
+
+```ts
+interface PersistQueryClientOptions {
+  /** The QueryClient to persist */
```

Comment:
I think this is still copy-pated from the persist plugin and should be `BroadcastQueryClientOptions` here :)

Reply:
Fixed 👍

Reply:
To answer your questions:
- **For now**, we are only syncing the cache within the clients, but similar to the persistence plugins, we could include other things as well like mutations and settings like you mentioned.
- Broadcast channels (and all of its fallback polyfills) are restricted by CORS, so this plugin is meant to sync between similar instance of the same app on the same subdomain/CORS. Anything more than that would require websockets and a common server connection, but that's a whole other topic :)